CGPs/cgp-0053.md

```diff
@@ -0,0 +1,100 @@
+---
+cgp: <to be assigned>
+title: Increase the gasLimit to 50M and gas targetDensity to 0.7
+date-created: 2022-04-03
+author: Mariano Cortesi (@mcortesi5), Javier Cortejoso (@jcortejoso), Gastón Ponti (@gastonponti)
+status: DRAFT
+discussions-to: <tbd>
+governance-proposal-id: <tbd>
+date-executed: <tbd>
+---
+## Overview
+
+CGP - Celo Governance Proposal
+
+This proposal sets the block gas limit parameter of the Celo blockchain from `20M` to `50M` and the gasPriceMinimum's targetDensity to `0.8`. The change is motivated by the improvement on block processing throughput with the latest celo-blockchain releases.
```

Comment:
i think typo here on ` gasPriceMinimum's targetDensity to 0.8` where it is 0.7

Reply:
Yes, you are right!